pkgs/development/libraries/science/math/cudnn/generic.nix

```diff
@@ -1,11 +1,11 @@
 {
-  stdenv,
+  backendStdenv,
```

Comment:
why do we need to accept `backendStdenv` here? couldn't we just pull it out of cudatoolkit or libcublas?

Reply:
`backendStdenv` being a top-level attribute of `cudaPackages`, I figured it makes sense. Plus, I hope we eventually drop `cudatoolkit`

Reply:
kook makes sense!